docs/assignation/README.md

```diff
@@ -1,99 +1,94 @@
+
+Once you have your tasks highlighted, select a user to assign and confirm your selection by clicking the **Confirm** button. Similarly, you can use the **unassign from selection** option to remove these assignments.
 
-When you finish all the assignations, quit this mode using the **Escape** key,
-or the cross on the top right corner of the action box.
+When you have finished all the assignments, exit this mode by pressing the **Escape** key or clicking the cross on the top right corner of the action box.
+<!-- I could get this to work, escape doesn't seem to do anything and I don't see an X to close the dialogue-->
 
-Now, you can see the artist's avatar next to each Status.
+Now, you can see the artist's avatar next to each of the task's status.
 
 ![Assigned people](../img/getting-started/assignation.png)
 
-To unclutter the list, you can hide this information with the **Hide
-assignations** button on the top of the spreadsheet ![Hide assignations](../img/getting-started/hide_assignation.png).
+You can toggle the visibility of task assignments on or off by using the **Show/Hide Assignments** button located at the top of the spreadsheet ![Hide assignations](../img/getting-started/hide_assignation.png).
 
-Once you assign someone, he is part of the production team. Depending
-In his position (Artist, Supervisor, Vendor), he only sees the production where
-he's assigned.
+Keep in mind that task assignments can determine what a user sees, depending on their role (for instance, vendors need to be assigned to a task before they can view anything).
 
-Now, everybody knows what to do and who is doing what!
+With your assignments completed, everyone knows their tasks and responsibilities!
 
 ## Assign tasks from the detailed Task Type page
 
-You can decide to assign tasks from the detailed task type page.
-The advantage is that you can focus on only one task type and have more information, such as the length of a shot, or focus on a specific asset type.
+You have the option to assign tasks directly from the detailed task type page. This allows you to focus on a single task type and access more detailed information, such as the length of a shot or specific details about an asset type.
+
+On the Detailed Task Type page, simply click on a status to open the comment panel.
 
-In the Detailed Task Type page, click on a status to open the comment panel.
+From there, you'll have the same options available as on the global page.
 
-From here, you have the same option as for the global page.
+You can select one or multiple tasks simultaneously to assign a user.
+<!-- This section needs re-writing as it is unclear. Is there referring to the page when you click on a specific shot or asset? There are multiple tabs (i.e info, chat, concepts, casting etc), and I'm assuming you are referring to all the tasks for said shot/asset on the info tab right? Good to also include a screenshot here-->
```

Comment:
It's about the list displayed when you click on a "task type column" header.

Reply:
Thanks, this has been updated in the latest commit